product-features/liquidity-providers/faqs.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Common questions you might have as a liquidity provider.
+---
+
+# FAQs
+
+<details>
+
+<summary>What is the difference between swap fee and fee rebate?</summary>
+
+The **swap fee** is the total fee charged to users for executing a token swap. It's the fee that directly impacts the trader and is displayed as "Fees" in the Pool Info panel.
+
+On the other hand, the **fee rebate** is the portion of the swap fee that is distributed to liquidity providers as a reward for supplying liquidity to the pool. The remaining portion of the swap fee goes to the ALEX platform. You can also find the fee rebate percentage in the Pool Info panel.
+
+![](../../.gitbook/assets/liquidity-providers/faqs-pool-info-panel-fees.png)
+
+</details>
+
+<details>
+
+<summary>What is the swap fee percentage that goes to liquidity providers?</summary>
+
+The swap fee percentage that goes to liquidity providers is known as the **fee rebate**. It is typically set at 50%, though it can vary depending on the pool. You can check this percentage in the Pool Info panel.
+
+</details>
+
+<details>
+
+<summary>What are the LP Tokens?</summary>
+
+Also know as "Pool Tokens", these tokens are issued to liquidity providers to represent their share of the liquidity pool. The total supply of LP tokens represents the 100% of the pool's funds.
+
+When users add liquidity to a pool, they receive LP tokens as proof of ownership. These tokens entitle them to a proportional share of the pooled assets and a portion of the fees generated by trades (swaps) within the pool.
+
+When liquidity is removed, the user transfers LP tokens back to the protocol. This determines how much of the pool's assets are returned to the user, along with their share of the transaction fees accrued during the time their liquidity was provided.
+
+</details>
+
+<details>
+
+<summary>How does the fee rebate benefit liquidity providers?</summary>
+
+The fee rebate is automatically reinvested into the pool, increasing the overall value of the pool. Since liquidity providers (LPs) hold a share of the pool, their holdings grow in value over time. However, these rewards can only be claimed when LP tokens (representing their share of liquidity) are withdrawn from the pool.
```

Comment:
The fee rebate is automatically accrued and reinvested into the pool, increasing the overall value of the pool. Since liquidity providers (LPs) hold a share of the pool, their holdings grow in value over time. However, these rewards can only be claimed when LP tokens (representing their share of liquidity) are withdrawn from the pool.

Reply:
Done